.env.template

```diff
@@ -4,5 +4,6 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=<FILL ME IN!!!>
-TRELLO_API_TOKEN=<FILL ME IN!!!>
\ No newline at end of file
+TRELLO_API_KEY=b4d785b62d3d959eaa53683db66cc449
+TRELLO_API_TOKEN=c9a9f0c3fe6cab86aeba80f8c75de6d6c839578f294e51a52cba87684c066f5d
```

Comment:
These look like real values! You shouldn't commit these to git, they should only be in your `.env` file which stays on your machine.

Please revoke this Trello token and make a new one: https://help.trello.com/article/1183-revoking-a-trello-token